src/QsCompiler/CompilationManager/EditorSupport/CodeActions.cs

```diff
@@ -348,6 +351,26 @@ where DiagnosticTools.ErrorType(ErrorCode.UnknownType)(diagnostic)
             var orSuggestions = diagnostics.Where(DiagnosticTools.WarningType(WarningCode.DeprecatedORoperator))
                 .Select(d => ReplaceWith(Keywords.qsORop.op, d.Range));
 
+#pragma warning disable 0618
```

Comment:
One minor thing: I would prefer putting this only on the two lines that actually need it.

Reply:
Thanks!